tests/AtlasConnectivity.Tests/AtlasConnectivity.Tests.csproj

```diff
@@ -45,6 +45,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="XunitXml.TestLogger" Version="2.1.26" />
```

Comment:
why somewhere you use `Xunit` and somewhere `Junit`? Should it be just `xunit`?

Reply:
The Xunit logger doesn't print out the correct format that Evergreen understands. The Junit logger does.

Reply:
(It's a bit misleading as the Evergreen task is attach.xunit but it really means Junit format.

Reply:
ok (it looks like we cannot do more here), but anyway let's be consistent and add the same logger everywhere